addons/l10n_ec/models/l10n_ec_sri_payment.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError, ValidationError
+import odoo.addons.decimal_precision as dp
+
+class SriPayment(models.Model):
```

Comment:
If this is only name and code, why do not we make it a selection field?   (too many codes)  Also, the string should be in English.